HumanControls/HumanControls.ino

```diff
@@ -36,17 +40,35 @@ MenuController menuController(ENCODER_PIN_CLK, ENCODER_PIN_DT, DISPLAY_ADDRESS,
                               ANGLE_INCREMENT, ANGLE_MIN, ANGLE_MAX, PRESSURE_INCREMENT, PRESSURE_MIN, PRESSURE_MAX,
                               DURATION_INCREMENT, DURATION_MIN, DURATION_MAX);
 
+EnableController enableController;
+FireController fireController;
+
+bool isConnected = false;
+String statuses[3] = {"disabled", "enabled", "primed"};
+String status = statuses[0];
```

Comment:
This will copy the string each time it changes, which is no bueno. It should be a reference instead.

Reply:
Done.